vm/systemSmartContracts/esdt.go

```diff
@@ -1442,6 +1446,10 @@ func (e *esdt) checkSpecialRolesAccordingToTokenType(args [][]byte, token *ESDTD
 		return validateRoles(args, e.isSpecialRoleValidForNonFungible)
 	case core.SemiFungibleESDT:
 		return validateRoles(args, e.isSpecialRoleValidForSemiFungible)
+	case metaESDT:
```

Comment:
some unit tests will be great since we have the config refactor feature pending and will help us confirm that everything is ok after the integration